src/Command/QuestionableCommand.php

```diff
@@ -23,18 +23,14 @@
 abstract class QuestionableCommand extends Command
 {
     /**
-     * @param string   $questionText
-     * @param mixed    $default
-     * @param callable $validator
-     *
      * @return mixed
      */
     final protected function askAndValidate(
         InputInterface $input,
         OutputInterface $output,
-        $questionText,
-        $default,
-        $validator
+        string $questionText,
+        string $default,
```

Comment:
before it was mixed 🤔

Reply:
I had trouble to see how it could be something else.
And, in our case, we only use it as string.